contrib_bots/lib/poll.py

```diff
@@ -0,0 +1,103 @@
+class PollHandler(object):
+    '''
+    This plugin allows users to create polls.
+    '''
+
+    def __init__(self):
+        # polls is a dict of dict. First dict corresponds to poll title,
+        # second refers to answers in poll
+        self.polls = dict()
+
+        # streams is a convenience object that keeps track of the latest
+        # question for each stream.
+        self.streams = dict()
+
+    def usage(self):
+        return '''
+        This plugin facilitates polling.
+
+        Commands:
+        @poll init <question>: <option1>, <option2>, <option3>
+        A valid poll must use a colon (:) to divide question from answers.
+        Answers must be seperate by coma (,).
+
+        @poll <option1>
+        Question will be infered by scanning channel for last poll with
+        valid answer.
+
+        @poll result poll title
+        Returns a formatted version of the poll.
+
+        @poll help
+        Returns usage for polling bot.
+
+        >>> @poll init what's the best ice cream flavour: vanilla, chocolate
+        <<< Question successfully created! Answer away
```

Comment:
Yup, I made the change. I forgot to update this though. 

![image](https://cloud.githubusercontent.com/assets/7867183/21435050/95495a1e-c81b-11e6-8231-1da4cc5dcd3e.png)